src/python_pybind11/CMakeLists.txt

```diff
@@ -62,6 +67,7 @@ if (${pybind11_FOUND})
   if (BUILD_TESTING)
     # Add the Python tests
     set(python_tests
+      Angle_TEST
```

Comment:
I don't see these tests running on CI :female_detective:

Reply:
This might happens on Bionic. We at least require `pybind11 2.2`, but Bionic installs 2.0.1-4 `Get:20 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python3-pybind11 all 2.0.1-4 [4652 B]`

Focal is fine https://github.com/ignitionrobotics/ign-math/runs/4559329091?check_suite_focus=true

maybe pip could fix this issue, I will check it

Reply:
Ah gotcha, thanks! I wouldn't worry about Bionic, targeting Focal is fine!